Sources/SwifterSwift/UIKit/UINavigationBarExtensions.swift

```diff
@@ -28,21 +28,28 @@ public extension UINavigationBar {
     ///
     /// - Parameter tint: tint color (default is .white).
     func makeTransparent(withTint tint: UIColor = .white) {
-        if #available(iOS 13.0, tvOS 13.0, *) {
+        let legacyAppearance = { [self] in
+            isTranslucent = true
+            backgroundColor = .clear
+            barTintColor = .clear
+            setBackgroundImage(UIImage(), for: .default)
+            titleTextAttributes = [.foregroundColor: tint]
+            shadowImage = UIImage()
+        }
+        #if os(tvOS)
+        legacyAppearance()
```

Comment:
@LucianoPAlmeida tvOS tests failed (actually they crashed) if you try to use the new appearance values.